src/commands/generateLicenseKey.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Make Positive Provar Ltd
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.md file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+import { messages } from '../messages';
+import { HttpService } from '../utils/httpService';
+
+const DEFAULT_VALIDATE_INPUT = (value: string) => {
+    return value === '' ? messages.generate_license_error_empty_field : null;
+};
+
+const VALIDATE_PHONE_INPUT = (value: string) => {
+    const regex = /^\d+$/;
+    return !regex.test(value) && value.length
```

Comment:
That's not a valid phone number regex.

Reply:
Ritesh, does this help at all?

https://github.com/catamphetamine/libphonenumber-js

@paulrichards-provar it's MIT licensed

Reply:
If there were a can of worms emoji I would post it. Basically you're shaving a yak here and I just wouldn't bother, google have solved it, use their MIT solution or a derivative based on it.  ☕  🐍 A cup of snakes is the best I can offer.

Reply:
> Ritesh, does this help at all?
> 
> https://github.com/catamphetamine/libphonenumber-js
> 
> @paulrichards-provar it's MIT licensed

@PeteProvar we can add this npm package